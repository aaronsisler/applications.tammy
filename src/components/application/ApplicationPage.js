import React from 'react';
import ApplicationProcessContainer from 'Application/ApplicationProcessContainer';

const ApplicationPage = (props) => (
    <div className="application_page">
        <ApplicationProcessContainer {...props} />
    </div>
)

export default ApplicationPage;
