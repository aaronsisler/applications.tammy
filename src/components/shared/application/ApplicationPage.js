import React from 'react';
import ApplicationProcessContainer from './ApplicationProcessContainer';

export default class ApplicationPage extends React.Component {
    constructor(props) {
        super(props);
    }

    componentDidMount() {
    }

    render() {
        return (
            <div id="application_page">
                <ApplicationProcessContainer />
            </div>
        );
    }
}
