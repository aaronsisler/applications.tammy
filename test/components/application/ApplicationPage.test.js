import React from 'react';
import { shallow } from 'enzyme';
import ApplicationPage from 'Application/ApplicationPage';

describe('ApplicationPage', () => {
    it('should render ApplicationPage correctly', () => {
        const wrapper = shallow(<ApplicationPage />);
        expect(wrapper).toMatchSnapshot();
    });
});
