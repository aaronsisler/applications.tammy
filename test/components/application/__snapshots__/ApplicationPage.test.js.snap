// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationPage should render ApplicationPage correctly 1`] = `
<div
  className="application_page"
>
  <Connect(ApplicationProcessContainer) />
</div>
`;
